src/main/java/uk/gov/hmcts/ccd/RestTemplateConfiguration.java

```diff
@@ -95,7 +95,11 @@ private HttpClient getHttpClient(final int timeout) {
         cm.setMaxTotal(maxTotalHttpClient);
         cm.closeIdleConnections(maxSecondsIdleConnection, TimeUnit.SECONDS);
         cm.setDefaultMaxPerRoute(maxClientPerRoute);
-        cm.setValidateAfterInactivity(validateAfterInactivity);
+
+        // Temporally 30 value until we can tune the real value then It will be set in the ENV variables.
```

Comment:
cm.setValidateAfterInactivity(30);   is the value from validateAfterInactivity attribute and the value defined in the values.yaml as HTTP_CLIENT_VALIDATE_AFTER_INACTIVITY. 

HTTP_CLIENT_VALIDATE_AFTER_INACTIVITY: '0'  --->  Defines period of inactivity in milliseconds after which persistent connections must
                                                  * be re-validated prior to being {@link #leaseConnection(java.util.concurrent.Future,
                                                   *   long, java.util.concurrent.TimeUnit) leased} to the consumer. Non-positive value passed
                                                   * to this method disables connection validation. This check helps detect connections
                                                   * that have become stale (half-closed) while kept inactive in the pool.